src/controllers/controllerpresetfilehandler.h

```diff
@@ -16,15 +16,17 @@ class ControllerPresetFileHandler {
     ControllerPresetFileHandler() {};
```

Comment:
I struggled to understand the purpose of this class and why it is separate from ControllerPreset. The documentation at the top of the file was only a little helpful. After looking around the code, it seems this class's role is to serialize/deserialize ControllerPreset objects to/from XML and save/load those files to/from the filesystem. Am I understanding that right?

Reply:
Yes. Added a comment.